ui/app/templates/components/job-editor.hbs

```diff
@@ -16,7 +16,7 @@
           Paste or author HCL or JSON to submit to your cluster, or select from a list of templates. A plan will be requested before the job is submitted. You can also attach a job spec by uploading a job file or dragging &amp; dropping a file to the editor.
         </p>
 
-        {{#if (can "write variable" path="*" namespace="*")}}
+        {{#if (can "read variable" path="nomad/job-templates/*" namespace="*")}}
```

Comment:
At first glance I thought this would get us into trouble, but the `write` policy for Variables is a little different than it is for anything else:
> Create or update Variables at this path. Includes the "list" capability but not the "read" or "destroy" capabilities.

:+1: